logzio/utils/parsers.go

```diff
@@ -67,3 +67,16 @@ func ParseStringToMapList(value string) []map[string]interface{} {
 
 	return returnObject
 }
+
+// ParseInterfaceSliceToStringSlice receives slice of interface, and returns a slice of string
+func ParseInterfaceSliceToStringSlice(interfaceSlice []interface{}) []string {
+	stringSlice := make([]string, 0, len(interfaceSlice))
+	for _, s := range interfaceSlice {
+		val, ok := s.(string)
+		if !ok {
+			val = ""
+		}
+		stringSlice = append(stringSlice, val)
```

Comment:
Any specific usages for an empty string in the slice?

Reply:
This way we keep the size of the slice as the original slice, and the empty strings will be ignored.